azure/functions/http.py

```diff
@@ -6,9 +6,13 @@
 
 from azure.functions import _abc as azf_abc
 from azure.functions import _http as azf_http
-
 from . import meta
 
+try:
+    from http.cookies import SimpleCookie
+except ImportError:
+    from Cookie import SimpleCookie  # type: ignore
```

Comment:
Which `http` module is this? I couldn't find he definition for SimpleCookie. Also, for Cookie as well.

Reply:
its the standard http python module (https://docs.python.org/3.8/library/http.cookies.html). For Cookie module it is a old module in python 2.7 https://docs.python.org/2/library/cookie.html. 

<img width="743" alt="image" src="https://user-images.githubusercontent.com/12449837/164533232-b072d70d-dda9-4ee4-b421-128de350ec2d.png">

Good point, python 2 is not supported, i can remove the except block.